README.org

```diff
@@ -8,13 +8,41 @@ The Scarf CLI is a set of tools aimed at empowering users and maintainers of ope
 Each of the Scarf commands uses the general abstractions provided by Nomia to identify their resources. For example, when adding a package to an environment with the Environment Manager, you would specify the package to add and the environment to add it to by name, using the common Nomia conventions for string representations of names. This provides a unified interface to specified disparate areas of software distribution and development. Now they can be seamlessly combined and efficiently realized, while also enabling integration with any other system that offers Nomia resources.
 * Environment Manager
 The Scarf Environment Manager is used to manage the environments for user sessions, development workflows, and automated systems. It is similar to a package manager, except in that it treats environments as first-class resources to be manipulated, combined, transferred, etc.
+
+* Get started
+
+** Before you begin
+
+To install and use the Scarf CLI, you must have [[https://nixos.org/download.html][Nix]] and [[https://git-scm.com/downloads][Git]] installed.
+
+** Install
+
+1. Clone this repository and switch to it. Run ~git clone https://github.com/scarf-sh/scarf.git && cd scarf~.
+2. Run ~./install~.
+
+** Try it
+
+You can test that Scarf works by creating and installing a package in a Scarf environment. Try this:
+
+1. Install Nix's Node.js package into your default user environment. Run ~scarf env add nixpkgs:node~.
+2. Compare your system with the Scarf environment. Run these commands:
+   
+   #+BEGIN_SRC sh
+     $ which node  // shows path to system Node.js, if any
+     $ scarf env enter which node  // shows path to Nix-provided Node.js
+   #+END_SRC
+
+3. Remove Nix's Node.js package from your environment. Run ~scarf env remove nixpkgs:node~.
+
 ** Usage
-All ~scarf env~ subcommands accept an ~--environment~ flag, to operate on/within a specified environment. The default is the user's global environment.
+
+~scarf~ has one subcommand, ~env~. All ~scarf env~ subcommands accept an ~--environment~ flag, to operate on/within a specified environment. The default is the user's global environment.
```

Comment:
Because the CLI only does the one thing right now, I transplanted the usage stuff here. In the future, as functionality expands, I'd expect this section to be replaced with something like "Next steps" which links off to detailed tasks and usage instructions for "manage environments", "query gateways", "turn straw into gold", and so on.